infrastructure/resource_group_level/main.bicep

```diff
@@ -200,14 +203,12 @@ var tfmApiAdditionalSecureConnectionStrings = {
 }
 
 var portalUiSettings = {
-    RATE_LIMIT_THRESHOLD: RATE_LIMIT_THRESHOLD
+    RATE_LIMIT_THRESHOLD: RATE_LIMIT_THRESHOLD // TODO:FN-1086 30 on dev, 10000 on feature
```

Comment:
Note that on dev, the rate limit is generally 300. If there is to be variations per-app, then we need to be cleverer here. Otherwise, we'll need up update the value of the secret to the desired one?